sources/packages/backend/libs/integrations/src/services/sfas/sfas-restriction.service.ts

```diff
@@ -108,6 +124,58 @@ export class SFASRestrictionService
     await this.repo.save(restriction, { reload: false, transaction: false });
   }
 
+  /**
+   * Updates processed to false for the SFAS restrictions corresponding to the provided SFAS Individual Id.
+   * @param individualId provided SFAS Individual Id.
+   */
+  async updateProcessedStatus(individualId: number): Promise<void> {
+    await this.repo.update({ individualId }, { processed: false });
```

Comment:
I do not see this simple update as essential to be part of this service.
Keeping it in the student service only and changing it as mentioned in the comment below will prevent many files from being changed along this PR because the dependency will no longer be needed for the student service.
https://github.com/bcgov/SIMS/pull/2910#discussion_r1509423750